tools/docker/README.md

```diff
@@ -4,19 +4,22 @@ Asterinas development Docker images are provided to facilitate developing and te
 
 ## Building Docker Images
 
-To build a Docker image for Asterinas and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
+Asterinas Docker image is based on an OSDK Docker image. To build a Docker image for Asterinas and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
 
 ```bash
 cd <asterinas dir>
 # Build Docker image
 docker buildx build \
     -f tools/docker/Dockerfile \
     --build-arg ASTER_RUST_VERSION=$(grep "channel" rust-toolchain.toml | awk -F '"' '{print $2}') \
+    --build-arg BASE_VERSION=${BASE_VERSION} \
     -t asterinas/asterinas:$(cat VERSION)-$(date +%Y%m%d) \
     .
 ```
 
-For the Intel TDX Docker image, it is based on a general Docker image. You can execute the following command:
+Here, `BASE_VERSION` represents the version of the OSDK Docker image you want to base it on.
+
+For the Intel TDX Docker image, it is based on an Asterinas Docker image. You can execute the following command:
```

Comment:
Revise this paragraph similar to my comments about the OSDK Docker image documentation.